azure-pipelines.yml

```diff
@@ -40,7 +41,7 @@ jobs:
         PYAMG_VERSION: '*'
         PILLOW_VERSION: '*'
         JOBLIB_VERSION: '*'
-        PYTEST_VERSION: '4.6.2'
```

Comment:
Do we never want it pinned to support older versions?

Reply:
Makes sense. PR updated to pin the version for this instance.